_episodes/XX-access-data.md

```diff
@@ -0,0 +1,329 @@
+---
+title: "Access satellite imagery using Python"
+teaching: TODO
+exercises: TODO
+questions:
+- "Where can I find open satellite data?"
+- "How do I query a STAC API using metadata filters?"
+- "How do I fetch remote raster datasets using Python?"
```

Comment:
> "How do I query a STAC API using metadata filters?"

I would rephrase to "How do I search for satellite imagery with the STAC API?" if this is the first introduction in the material to STAC. This makes it clearer what a STAC API is used for. I think this is necessary because most participants won't know what an API is and will be unfamiliar with STAC.

Reply:
Good point, I'll fix this!